include/rmm/exec_policy.hpp

```diff
@@ -54,8 +57,8 @@ class exec_policy : public thrust_exec_policy_t {
    * @param stream The stream on which to allocate temporary memory
    * @param mr The resource to use for allocating temporary memory
    */
-  explicit exec_policy(cuda_stream_view stream             = cuda_stream_default,
-                       rmm::mr::device_memory_resource* mr = mr::get_current_device_resource())
+  explicit exec_policy(cuda_stream_view stream           = cuda_stream_default,
+                       rmm::device_async_resource_ref mr = rmm::mr::get_current_device_resource())
```

Comment:
This actually qualifies those. Should I remove the superfluous `rmm::` prefix?

Reply:
Sure if we are in the namespace no need to qualify it.